packages/cf-style-const/src/variables.js

```diff
@@ -60,5 +71,6 @@ export default {
   colorPurpleLight: lighten(colorOffsetLight, '#d91698'),
   colorPurpleDark: darken(colorOffsetDark, '#d91698'),
   colorTwitterBlue: '#00aced',
-  colorFacebookBlue: '#3b5998'
+  colorFacebookBlue: '#3b5998',
+  mainBackground: '#e6e6e6'
```

Comment:
Renamed to `colorMainBackground`. I think we need to combine both approaches since if you want to easily change the backgroundColor, the same constant should not be used for a shadow around inputs.